diff --git a/majiq/rna_majiq/src/rna_majiq/_run/heterogen.py b/majiq/rna_majiq/src/rna_majiq/_run/heterogen.py
index 7c86004..552db62 100644
--- a/majiq/rna_majiq/src/rna_majiq/_run/heterogen.py
+++ b/majiq/rna_majiq/src/rna_majiq/_run/heterogen.py
@@ -236,7 +236,11 @@ def run(args: HeterogenArgs):
     (
         df
         # pull out and rename useful values from stats extra (at the time of writing, only tnom score)
-        .drop((f"{k}-stats_extra" for k, v in STATS_EXTRA_DEFS.items() if v is None and k in use_stats), axis=1)
+        .drop(
+            (f"{k}-stats_extra" for k, v in STATS_EXTRA_DEFS.items() if v is None and k in use_stats),
+            axis=1,
+            errors="ignore",
+        )
         .rename({f"{k}-stats_extra": v for k, v in STATS_EXTRA_DEFS.items() if v is not None and k in use_stats}, axis=1)
         # any column with pvalue in it needs to be manually formatted
         .pipe(
